browser/css/color-palette-dark.css

```diff
@@ -66,4 +66,4 @@
 [data-bg-theme='dark'] {
 	--color-cursor-blink-background: #fff;
 	--color-background-document: #121212;
-}
```

Comment:
would be good to don't have this change in (as it seems to be just an automatic editor change), but let's not block this PR because of this